test/functional/sqs-listener.spec.ts

```diff
@@ -0,0 +1,116 @@
+import AWS from "aws-sdk";
+import fs from "fs";
+import path from "path";
+
+import { configManager } from "../../src/services/config-manager";
+import { defaultConfig } from "../../src/config/default";
+import {
+  createConfigFromArgs,
+  createConfigFromEnv,
+} from "../../src/utils/config-utils";
+
+jest.setTimeout(60000);
+
+// Load the configuration for this service with the following precedence...
+//   process args > environment vars > config file.
+configManager.apply(defaultConfig);
+configManager.apply(createConfigFromEnv(process.env));
+configManager.apply(createConfigFromArgs(process.argv));
+
+AWS.config.update({
+  region: configManager.get("awsSqsRegion"),
+  accessKeyId: configManager.get("awsSqsAccessKey"),
+  secretAccessKey: configManager.get("awsSqsSecretAccessKey"),
+});
+const s3 = new AWS.S3({
+  endpoint: configManager.get("awsEndPoint"),
+  apiVersion: "2006-03-01",
+  signatureVersion: "v4",
+  s3ForcePathStyle: true,
+});
+
```

Comment:
I think the above would make more sense in either beforeEach or beforeAll, but I'm not very familiar with testing aws code

Reply:
It doesn't need to change in between tests. Should be fine.